index.html

```diff
@@ -20,10 +20,70 @@
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <!--
+    <header>
+      <ul class="header">
+        <h1 class="logo">Fitzone</h1>
+        <h2 class="nav1">HOME</h2>
+        <h2 class="nav2">ABOUT US</h2>
+        <h2 class="nav3">MEMBERSHIP</h2>
+        <h2 class="nav4">LOCATIONS</h2>
+        <h2 class="signup">SIGN UP</h2>
+        <h2 class="login">LOGIN</h2>
+      </ul>
+    </header>
+    <main>
+      <div class="banner">
+        <p class="p1">
+          Fitzone is commited to providing the best possible<br> 
+          experience that a gym can offer. We provide our<br>
+          members with access to high quality equipment, as well<br>
+          as a welcoming and judgement free environment designed<br>
+          to allow you to improve yourself at a pace that's right for you.<br>
+          Browse our membership options and begin your journey to be <br>
+          the best you.
+        </p>
+        <button type="submit" class="b1">BECOME A MEMBER</button>
+        <form class="info">
+          <label for="name">Name:</label>
+          <input type="text" id="fullname" name="fullname" autocomplete="name" />
+          <label for="email">Email:</label>
+          <input type="text" id="email" name="email" autocomplete="email" />
+        </form>
```

Comment:
Missing a way to actually `submit` the form. 

This can be done by adding either an `<input type="submit" />` or by adding a `button` element with `type="submit"`